homeassistant/components/upnp/sensor.py

```diff
@@ -149,6 +158,11 @@ class UpnpSensor(UpnpEntity, SensorEntity):
 class RawUpnpSensor(UpnpSensor):
     """Representation of a UPnP/IGD sensor."""
 
+    @property
+    def device_class(self):
+        """Return the state class for the sensor."""
+        return self.entity_description.state_class
+    
```

Comment:
The property should not be needed. It should be automatically picked up from the `entity_description` by the base class.
https://github.com/home-assistant/core/blob/baa810aabbee430643229ee4b3e9f4c467d2878e/homeassistant/components/sensor/__init__.py#L301-L302

```suggestion
```